components/steering_wheel/src/IO/IO.c

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
A whole C file just to make a global? Not worth it, unless there's a benefit I'm not seeing here

Reply:
Has to do with a previous comment you made about making a global struct that has all of the IO data aggregated. It was either splitting IO_Digital, IO_Analog, and having and aggregate IO or mashing everything into a single IO, and I thought that this method would be cleaner. What do you think about the options?

Reply:
yeah but then I commented that it was a bad idea lol. What's the benefit of this? `IO->Analog` vs `IO_Analog`. Second one has one less character and takes up less memory

Reply:
Ahh I didnt remember the second comment. In that case I will return it to IO_A and IO_D as it was